### PR TITLE
Initialised gems and models

### PR DESCRIPTION
**Pull request by lbreen**
Added models

### CODE REVIEW

File: db/migrate/20160822152448_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email_address
+      t.string :email_address
```
Comment (disha-p) [inline db/migrate/20160822152448_create_users.rb]: why do we have 2 email addresses in here?

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```
Comment (disha-p) [inline app/models/user.rb:2]: add has_many relationships

File: db/migrate/20160822152448_create_users.rb (hunk 1 above)
Comment (lbreen) [inline db/migrate/20160822152448_create_users.rb]: typo - changed line 7 to postal_address

File: app/models/user.rb (hunk 2 above)
Comment (lbreen) [inline app/models/user.rb:2]: will do when we break into our individual tasks

File: db/migrate/20160822152810_create_menus.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class CreateMenus < ActiveRecord::Migration[5.0]
+  def change
+    create_table :menus do |t|
+      t.references :user, foreign_key: true
+      t.string :name
+      t.string :photo
+      t.integer :price
+      t.integer :preparation_time
+      t.integer :min_guests
+      t.integer :mex_guests
```
Comment (disha-p) [inline db/migrate/20160822152810_create_menus.rb]: max_guests

**Pull request merged**